docs/section4/4-4.md

```diff
@@ -0,0 +1,34 @@
+# Airside Systems – Rule 4-4
+
+**Rule ID:** 4-4
+**Rule Description:** Where no heating and/or cooling system is to be installed, and a heating or cooling system is being simulated only to meet the requirements described in Section G3.1-10 HVAC Systems proposed column c and d, heating and/or cooling system fans shall simulated to be cycled ON and OFF to meet heating and cooling loads during all hours.  
```

Comment:
The logic only checks for fan operation during occupied hours. Please update rule description accordingly, because now it suggests that both occupied and unoccupied hours are verified.